api.web.yaml

```diff
@@ -144,6 +144,13 @@ paths:
   /users:
     get:
       description: Get all the users a user has access to view
+      parameters:
+        - in: query
+          name: find
+          required: false
+          description: user details to lookup by
```

Comment:
better description and some examples here are probably warranted

Reply:
https://github.com/ucl-arc-tre/portal/pull/251/commits/58c314f75f8821f74fb6d94cad5ed6f13fd5d1e2